Objects/typeobject.c

```diff
@@ -8064,7 +8064,7 @@ static slotdef slotdefs[] = {
     UNSLOT("__abs__", nb_absolute, slot_nb_absolute, wrap_unaryfunc,
            "abs(self)"),
     UNSLOT("__bool__", nb_bool, slot_nb_bool, wrap_inquirypred,
-           "self != 0"),
+           "True if self else False"),
```

Comment:
Isn’t this a circular definition, since ‘if self’ uses bool?

Reply:
Maybe "Determine the truth value of self"? (Referencing https://docs.python.org/3/library/stdtypes.html#truth-value-testing)

Reply:
It's circular, but that seems to be the style for most of these (see the abs one above it for example). Though if you go up and down a bit further, there's others that do go into more detail. For example, `__len__` is `"__len__($self, /)\n--\n\nReturn len(self)."`.

Reply:
I propose:

```suggestion
           "self is truthy"),
```

Reply:
The PR's current wording is closer to the style of the other docstrings in this file. Maybe we should rewrite them all but I'd rather just fix the clearly incorrect wording for this method.

Reply:
@merwok that's incorrect. None is falsey but does not `== False`.

Reply:
Irit is correct.  The expansion "bool(self) = True if (bool(self) is True) else False" is consistent but uninformative.